common/beerocks/bwl/include/bwl/sta_wlan_hal.h

```diff
@@ -48,6 +48,8 @@ class sta_wlan_hal : public virtual base_wlan_hal {
                          bool mem_only_psk, const std::string &bssid, uint8_t channel,
                          bool hidden_ssid) = 0;
 
+    virtual bool associate(const sMacAddr &bssid, uint8_t channel) = 0;
```

Comment:
I just noticed now: the roam() function, three lines below, is in fact exactly what we need, so this commit is not needed at all.

I've removed the Ready for Merge label so you can remove this commit.

Reply:
removed useless commit